recipes/spiceypy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "spiceypy" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "33db27531f7e8e31ce06970231b3265fe043b608d1404c9f43b2af14508713f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.zip
+  url: https://github.com/AndrewAnnex/SpiceyPy/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+    - 0001-Remove-CSPICE-installation-logic.patch
```

Comment:
What is the purpose of this patch?

Reply:
spicepy has a bit of code in setup.py that builds a shared library from cspice source during installation. given the cspice conda-forge recipe that logic won't be needed.